firmware/user/modes/mode_snake.c

```diff
@@ -864,7 +864,10 @@ void ICACHE_FLASH_ATTR snakeMoveSnake(void)
         snake.score += snake.scoreMultiplier;
 
         // Play a jingle
+        led_t leds[NUM_LIN_LEDS] = {{0}};
+        setLeds(leds, sizeof(leds));
         startBuzzerSong(&foodSfx);
+        setLeds(leds, sizeof(leds));
```

Comment:
you don't need to turn off the LEDs twice

Reply:
TODO: replace this with an actual blink instead of fake-blinking by briefly turning the LEDs off.

Reply:
Cleaned up this code though.